uwblueprint/sistering
# Create volunteer shifts page component

## Ticket link

Closes #128 



## Implementation description
* Create a page in `frontend\src\components\pages\volunteer\shifts` called `VolunteerShiftsPage.tsx` with the route `/volunteer/shifts`.



## Steps to test
1. Go to `http://localhost:3000/volunteer/shifts` and verify that the volunteer shifts page loads properly.



## What should reviewers focus on?
* No silly mistakes 😄 


## Checklist
